### PR TITLE
User group update

### PR DESCRIPTION
**Pull request by partydragen**

### CODE REVIEW

**Comment by samerton** [comment]
So far so good!

File: core/templates/frontend_init.php [hunk 1]
```diff
@@ -71,7 +71,7 @@
 
 		$cache->store('default_group', $default_group);
 	}
-	if($user->data()->group_id == $default_group && ($user->data()->reset_code)){
+	if(in_array($default_group, $user->getAllGroupIds) && ($user->data()->reset_code)){
```
[the remark this answers is not in the document]
Comment (partydragen) [reply core/templates/frontend_init.php:74]: > getAllGroupIds is a function right? Looks like you're missing the ()

Ohhh look at that, weird that did not cause a error when i loaded the website

**Comment by Derkades** [comment]
:shipit:

**Comment by Aberdeener** [comment]
this done @partydragen?
I think derk needs it for bot testing, then we can ship pre8

**Comment by Derkades** [comment]
samerton needs to review (probably just merge, who is going to look at 4000+ lines?)

**Comment by Derkades** [comment]
merge now, review later :1st_place_medal:

**Pull request merged**